INFO,runarguments,MainThread,function,Line: 88,2015-01-20 14:57:32,[1m**** Starting The APE ****[0;0m
DEBUG,fetcharguments,MainThread,function,Line: 97,2015-01-20 15:00:27,Getting Plugin: Ape
DEBUG,quartermaster,MainThread,get_plugin,Line: 76,2015-01-20 15:00:27,Retrieving Ape
ERROR,crash_handler,MainThread,log_error,Line: 38,2015-01-20 15:00:27,[1m********** Oops, I Crapped My Pants **********[0;0m
ERROR,crash_handler,MainThread,log_error,Line: 51,2015-01-20 15:00:27,[31m[1mImportError: [0;0m[31mEntry point ('ape.plugins', 'plugins') not found[0;0m
ERROR,crash_handler,MainThread,log_error,Line: 53,2015-01-20 15:00:27,[31m[1mFailed Line: [0;0m[31mraise ImportError("Entry point %r not found" % ((group, name),))[0;0m
ERROR,crash_handler,MainThread,log_error,Line: 55,2015-01-20 15:00:27,[31m[1mIn Function: [0;0m[31mload_entry_point[0;0m
ERROR,crash_handler,MainThread,log_error,Line: 57,2015-01-20 15:00:27,[31m[1mIn File: [0;0m[31mpkg_resources.py[0;0m
ERROR,crash_handler,MainThread,log_error,Line: 59,2015-01-20 15:00:27,[31m[1mAt Line: [0;0m[31m2562[0;0m
ERROR,crash_handler,MainThread,log_error,Line: 60,2015-01-20 15:00:27,[1m**********************************************[0;0m
DEBUG,crash_handler,MainThread,log_error,Line: 61,2015-01-20 15:00:27,Traceback (most recent call last):
  File "/home/charon/repositories/code/hortators/theape/theape/infrastructure/crash_handler.py", line 15, in wrapped
    return method(self, *args, **kwargs)
  File "/home/charon/repositories/code/hortators/theape/theape/infrastructure/arguments/fetcharguments.py", line 99, in function
    plugin = self.quartermaster.get_plugin(name)
  File "/home/charon/repositories/code/hortators/theape/theape/plugins/quartermaster.py", line 78, in get_plugin
    return self.plugins[name]
  File "/home/charon/repositories/code/hortators/theape/theape/plugins/quartermaster.py", line 50, in plugins
    self._plugins = self.import_plugins()
  File "/home/charon/repositories/code/hortators/theape/theape/infrastructure/ryemother.py", line 78, in __call__
    return self.from_entry_point(parent, group, name, keyfunction)
  File "/home/charon/repositories/code/hortators/theape/theape/infrastructure/ryemother.py", line 129, in from_entry_point
    module = pkg_resources.load_entry_point(self.base_package, group, name)
  File "/home/charon/.python27/theape/local/lib/python2.7/site-packages/pkg_resources.py", line 468, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/home/charon/.python27/theape/local/lib/python2.7/site-packages/pkg_resources.py", line 2562, in load_entry_point
    raise ImportError("Entry point %r not found" % ((group, name),))
ImportError: Entry point ('ape.plugins', 'plugins') not found

